code/obj/machinery/manufacturer.dm

```diff
@@ -1811,8 +1786,8 @@
 			var/remove_link = ""
 			var/pause_link = ""
 			if (queue_num == 1)
-				// if (istype(A,/datum/manufacture/) && src.speed != 0 && timeleft != 0)
-				// 	time_number = round(src.timeleft / src.speed)
+				// if (istype(A,/datum/manufacture/) && src.speed != 0 && time_left != 0)
+				// 	time_number = round(src.time_left / src.speed)
```

Comment:
```suggestion
```
may as well deleted commented code as long as it's nothing that's been worked on very recently

Reply:
/very/ is emphasized here

Reply:
it's been close to two years, so out she goes